_posts/2022-03-23-blog-newsletter-social-media.md

```diff
@@ -0,0 +1,70 @@
+
+- In these informal communications, use a less technical language, adapt your writing style to the audience and make it accessible to non-experts. Avoid jargon and abbreviations that are common within your discipline but not known outside your field of research. 
+
+- Ensure the information shared is referenced correctly and that it is licensed under a [CC BY license][3] that dictates how you and others are allowed to use and share the material. 
+
+    For more information, please referr to the guideline on [Licensing]()
+
+- Make sure all contributors are attributed fairly. This increases in importance the more comprehensive the material shared, e.g. more important in a blog post or newsletter than a tweet. In shorter formats, it is good practice to attribute to the contributors by adding links to their profiles on the social media platform of choice (e.g. LinkedIn, Twitter, Facebook). Attributing contributors there also helps to gain and provide 
+visibility. 
+
+    For more information, please refer to the guideline on [Attribution]()
+
+- This is digital communication. Therefore, use hyperlinks to guide your readers to more information, and always to the original articles and open science outputs. 
+
+    For more information, please refer to the guideline on [Referencing]()
+
+- Lastly, you don’t have to wait until the end to communicate. While journal articles report on completed research output, social media communication can be used to communicate during the project, to solicit stakeholders interest, and to increase collaboration. 
+
+    *Be careful though, to not disclose scientific contributions before they are published as formal academic publications!*
+
+![BlogNewsletterSocialMedia_diagram](img/BlogNewsletterSocialMedia.jpg)
+
+## Specific pointers for different types of communication
+### Newsletter and blog posts
+-	In a ***newsletter***, summarize the research ‘news’ in a [Lay Summary][1]. Link and reference to the original work, and give attributions to contributors. 
+-	In a ***blog post***, you can write a longer popular science summary of the research, or describe some aspects of it – e.g. a case example, an impact it is already showing, or a story on where and how it was presented. Link and reference to the original work, and give attributions to contributors. 
+-	In *both newsletters and blogs*: ensure there is continuity. A blog with only one blog post is less efficient in communication than a blog where new blog posts are added every so often. The same goes for newsletters. It may be better to find or create a newsletter or blog with a broader scope, if news are regularly posted there. Include blog maintenance time in your plan to ensure that comments are answered and new blog posts/newsletters are published at the frequency you want. 
```

Comment:
```suggestion
-	In *both newsletters and blogs*: ensure there is continuity. A blog with only one blog post is less efficient in communication than a blog where new blog posts are added every so often. The same goes for newsletters. It may be better to find or create a newsletter or blog with a broader scope, if news are regularly posted there. Include blog maintenance time in your plan to ensure that comments are answered and new blog posts/newsletters are published at the frequency you want. 
```
Not sure if this section is relevant for the guideline. I would suggest to remove it.
@willu47 what do you think?